mars/services/task/execution/ray/executor.py

```diff
@@ -527,3 +529,86 @@ def _get_subtask_output_keys(chunk_graph: ChunkGraph):
             else:
                 output_keys[chunk.key] = 1
         return output_keys.keys()
+
+    async def _update_progress_and_collect_garbage(
+        self, subtask_graph: SubtaskGraph, interval_seconds: float
+    ):
+        object_ref_to_subtask = self._cur_stage_first_output_object_ref_to_subtask
+        total = len(subtask_graph)
+        completed_subtasks = OrderedSet()
+
+        deletion_seq = []
```

Comment:
We don't need the `deletion_seq`, a mock `_task_context` can be used for testing.

Reply:
Thanks for your reminder. I'll fix it.